pom.xml

```diff
@@ -1,169 +0,0 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-
-    <modelVersion>4.0.0</modelVersion>
-    <groupId>com.github.kstyrc</groupId>
-    <artifactId>embedded-redis</artifactId>
-    <packaging>jar</packaging>
-    <version>0.7-SNAPSHOT</version>
-    <name>embedded-redis</name>
-    <description>Redis embedded server for Java integration testing</description>
-    <url>https://github.com/kstyrc/embedded-redis</url>
-
-    <licenses>
-        <license>
-            <name>The Apache Software License, Version 2.0</name>
-            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
-            <distribution>repo</distribution>
-        </license>
-    </licenses>
-
-    <scm>
-        <url>https://github.com/kstyrc/embedded-redis</url>
-        <connection>scm:git:https://github.com/kstyrc/embedded-redis.git</connection>
-        <developerConnection>scm:git:https://github.com/kstyrc/embedded-redis.git</developerConnection>
-        <tag>embedded-redis-0.5</tag>
-    </scm>
-
-    <developers>
-        <developer>
-            <name>Krzysztof Styrc</name>
```

Comment:
we should add an AUTHORS file including Krzysztof and also a top level LICENSE.txt file as in the other repos

Reply:
Ok done. Should we add the other authors mentioned in the README.md?

Reply:
seems reasonable to me, yeah